ddsp/core.py

```diff
@@ -208,12 +208,21 @@ def gradient_reversal(x):
 
 
 # Unit Conversions -------------------------------------------------------------
-def midi_to_hz(notes: Number, zero_silence: bool = False) -> Number:
-  """TF-compatible midi_to_hz function."""
+def midi_to_hz(notes: Number, midi_zero_silence: bool = False) -> Number:
+  """TF-compatible midi_to_hz function.
+
+  Args:
+    notes: Tensor containing encoded pitch in MIDI scale.
+    midi_zero_silence: Whether to output 0 hz for midi 0, which would be convenient
+    when midi 0 represents silence.
+
+  Returns:
+    hz: frequency of MIDI in hz
```

Comment:
Full sentence please, capatalize and period

Reply:
Got it! Thanks so much for pointing it out!

Reply:
Changed!